cds/modules/deposit/static/templates/cds_deposit/angular-schema-form/uiselect.html

```diff
@@ -21,6 +21,7 @@
                 {{select_model.selected.name}}
             </ui-select-match>
       <ui-select-choices
+        minimum-input-length="3"
```

Comment:
i tried to set `minimum-input-length="form.minLength || 3"` but it was not working